packages/neos-ui-ckeditor5-bindings/src/EditorToolbar/TableDropDown.js

```diff
@@ -40,12 +41,13 @@ export default class TableDropDownButton extends PureComponent {
     }
 
     render() {
+        const iconDataUri = svgToDataUri(ckeIcons[this.props.icon]);
         return (
             <DropDown
                 padded={false}
             >
                 <DropDown.Header title={this.props.i18nRegistry.translate(this.props.tooltip)}>
-                    <img style={{verticalAlign: 'text-top'}} src={ckeIcons[this.props.icon]} alt={this.props.i18nRegistry.translate(this.props.tooltip)} />
+                    <img style={{verticalAlign: 'text-top'}} src={iconDataUri} alt={this.props.i18nRegistry.translate(this.props.tooltip)} />
```

Comment:
öhm do i get that right that we want to have the svgs as data-url and not as raw content string?
That can be done via the build setup. For the neos ui i introduced the convention that files ending with `.dataurl.svg` will get that treatment via the build setup already introduced via:

https://github.com/neos/neos-ui/pull/3836

Reply:
The issue here was that the content of `this.props.icon` is just the SVG markup.
And that in a src tag is not working, and I bet that the build pipeline also doesn't handle it.